lib/test-worker.js

```diff
@@ -1,38 +1,56 @@
 'use strict';
-/* eslint-disable import/order */
-const isObj = require('is-obj');
-const process = require('./process-adapter');
 
-const opts = process.opts;
-const testPath = opts.file;
+// Check if the test is being run without AVA cli
+{
+	/* eslint-disable import/order */
+	const path = require('path');
+	const chalk = require('chalk');
+
+	const isForked = typeof process.send === 'function';
```

Comment:
I am pretty sure moving this in here will break `karma-ava`. Anything that relies on the Node environment (vs the browser) should remain in `process-adapter`.

`process-adapter` is swapped out by `karma-ava` when compiling for the browser with an implementation that polyfills any missing functionality.